### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> **Note**
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ updates:
       - dependency-name: actions/setup-python
       - dependency-name: hashicorp/setup-terraform
       - dependency-name: mxschmitt/action-tmate
+      # # Managed by cisagov/skeleton-python-library
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/upload-artifact
```
Comment (jsf9k) [inline .github/dependabot.yml:24]: I think these lines should be uncommented:
```suggestion
      # Managed by cisagov/skeleton-python-library
      - dependency-name: actions/download-artifact
      - dependency-name: actions/upload-artifact
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
I have one minor change to request.

**Review by dav3r — APPROVED** [review]

**Review by jsf9k — CHANGES_REQUESTED** [review]
[The issue I brought up in my original review](https://github.com/cisagov/terraform-to-secrets/pull/2/files#r1226974260) still stands.

File: .github/dependabot.yml (hunk 1 above)
Comment (jsf9k) [reply .github/dependabot.yml:24]: Ah, this is one where the repo still has a `first-commits` branch.

**Review by jsf9k — APPROVED** [review]

**Pull request merged**